src/org/folio/rest/Okapi.groovy

```diff
@@ -208,6 +208,43 @@ class Okapi extends GeneralParameters {
             throw new AbortException("Error during Elastic Search reindex." + http.buildHttpErrorMessage(res))
         }
     }
+
+    def configureLdpDbSettings(tenant, admin_user, db_host, db_name, db_user, db_password ) {
+        auth.getOkapiToken(tenant, admin_user)
+        String url = okapi_url + "/ldp/config/dbinfo"
+        ArrayList headers = [
+            [name: 'Content-type', value: "application/json"],
+            [name: 'X-Okapi-Tenant', value: tenant.getId()],
+            [name: 'X-Okapi-Token', value: tenant.getAdminUser().getToken() ? tenant.getAdminUser().getToken() : '', maskValue: true]
+        ]
+        logger.info("Starting Configure LDP DB settings")
+        String body = """{"value":"{\\"pass\\":\\"${db_password}\\",\\"user\\":\\"${db_user}\\",\\"url\\":\\"jdbc:postgresql://${db_host}/${db_name}\\"}","tenant":"${tenant.getId()}","key":"dbinfo"}"""
+        def res = http.putRequest(url, body, headers)
+        if (res.status == HttpURLConnection.HTTP_OK) {
+            return tools.jsonParse(res.content).id
+        } else {
+            throw new AbortException("Error during Configure LDP DB settings" + http.buildHttpErrorMessage(res))
+        }
+    }
+
+    def configureLdpSavedQueryRepo(tenant, admin_user, ldp_queries_gh_owner, ldp_queries_gh_repo, ldp_queries_gh_token) {
+        auth.getOkapiToken(tenant, admin_user)
+        String url = okapi_url + "/ldp/config/sqconfig"
+        ArrayList headers = [
+            [name: 'Content-type', value: "application/json"],
+            [name: 'X-Okapi-Tenant', value: tenant.getId()],
+            [name: 'X-Okapi-Token', value: tenant.getAdminUser().getToken() ? tenant.getAdminUser().getToken() : '', maskValue: true]
+        ]
+        logger.info("Starting Configure Saved Query repo")
+        String body = """{"key": "sqconfig","tenant":"${tenant.getId()}","value":"{\\"owner\\":\\"${ldp_queries_gh_owner}\\",\\"repo\\":\\"${ldp_queries_gh_repo}\\",\\"token\\":\\"${ldp_queries_gh_token}\\"}"}"""
```

Comment:
Maybe better to use template engine here?

Reply:
Reworked with template engine.
@OHaimanov Please check now.